src/js/modules/infragistics.datasource.js

```diff
@@ -4359,8 +4372,8 @@
 				/*A.T. workaround for jQuery's 1.5 and above bug related to dataFilter and success callback.
 				We need to explicitly set the dataType to "text" when manually parsing it */
 				/* get jquery version */
-				if ($.fn.jquery) {
-					ver = $.fn.jquery.split(".");
+				if (jQuery.fn.jquery) {
```

Comment:
Yep, it should be used $ instead of jQuery. I didn't get latest version and I made a pull request without latest changes that @skrustev did. I just committed with the appropriate use of $ :)